addons/hr_holidays/static/src/dashboard/time_off_card.xml

```diff
@@ -0,0 +1,70 @@
+        <t t-set="duration" t-value="props.requires_allocation?props.data['virtual_remaining_leaves']:props.data['virtual_leaves_taken']" />
+        <t t-set="show_popover" t-value="true" />
+
+        <strong class="o_timeoff_name"><t t-esc="props.name"/></strong>
+        <span class="o_timeoff_duration">
+            <t t-if="props.data['icon'] !== false">
+                <img t-att-src="props.data['icon']" />
+            </t>
+            <t t-esc="duration"/>
+        </span>
+        <div class="text-uppercase">
+            <t t-if="props.data['request_unit'] == 'hour'">hours</t>
+            <t t-else="">days</t>
+            <t t-if="props.requires_allocation" name="duration_type">
+                available
+            </t>
+            <t t-else="">
+                taken
+            </t>
+            <t t-if="show_popover">
+                <TimeOffCardPopover
+                    allocated="props.data['max_leaves']"
+                    approved="props.data['leaves_approved']"
+                    planned="props.data['leaves_requested']"
+                    left="props.data['virtual_remaining_leaves']" />
+            </t>
+        </div>
+        <span t-if="props.requires_allocation and props.data['closest_allocation_expire'] !== false" class="text-uppercase o_timeoff_validity">
+            <t t-if="props.data['closest_allocation_remaining'] != props.data['virtual_remaining_leaves']">
+                (<t t-esc="props.data['closest_allocation_remaining']"/> <t t-if="props.data['request_unit'] == 'hour'">hours</t><t t-else="">days</t>
+                valid until <span t-esc="props.data['closest_allocation_expire']"/>)
+            </t>
+            <t t-else="">
+                (valid until <span t-esc="props.data['closest_allocation_expire']"/>)
+            </t>
+        </span>
+    </div>
+
+    <t t-name="hr_holidays.TimeOffCardMobile" owl="1">
+        <t t-set="duration" t-value="props.requires_allocation?props.data['virtual_remaining_leaves']:props.data['virtual_leaves_taken']" />
+
+        <span class="pull-right o_timeoff_card_mobile">
+            <t t-if="props.requires_allocation" name="duration_type">
+                <strong t-esc="duration" class="o_timeoff_green"/> / <span t-esc="props.data['max_leaves']"/> <t t-if="props.data['request_unit'] == 'hour'">Hours</t><t t-else="">Days</t> <span class="o_timeoff_green">Available</span>
+            </t>
+            <t t-else="">
+                <strong t-esc="duration"/> <t t-if="props.data['request_unit'] == 'hour'">Hours</t><t t-else="">Days</t> <span class="o_timeoff_purple">Taken</span>
+            </t>
+        </span>
+    </t>
+
+    <t t-name="hr_holidays.TimeOffCardPopover" owl="1">
+        <div class="o_timeoff_info">
+            <Popover position="'right'" popoverClass="'o_timeoff_popover'">
+                <span class="fa fa-question-circle-o"/>
+                <t t-set="opened">
```

Comment:
What does this `opened` variable do? I found in the codebase some usages of it, but didn't understand where it's used

Reply:
It's the slot for the body of the popover :nerd_face:  (took me a while to figure it out)
https://github.com/odoo/odoo/blob/master/addons/web/static/src/legacy/xml/base.xml#L288